src/locale/test-keymap-util.c

```diff
@@ -190,12 +190,17 @@ static void test_x11_convert_to_vconsole(void) {
 }
 
 int main(int argc, char **argv) {
+        _cleanup_free_ char *map = NULL;
+
         test_setup_logging(LOG_DEBUG);
 
         test_find_language_fallback();
         test_find_converted_keymap();
         test_find_legacy_keymap();
 
+        assert_se(get_testdata_dir("test-keymap-util/kbd-model-map", &map) >= 0);
+        assert_se(setenv("SYSTEMD_KBD_MODEL_MAP", map, 1) == 0);
```

Comment:
```C
    if (!getenv(SYSTEMD_KBD_MODEL_MAP"))
              /* We are not running from meson, let's skip the tests which might be unreliable with a kbd-model-map from the system. */
              return 0;
```
?

Reply:
Well since this patch allows the test to run anywhere and the changes are pretty trivial, I don't think this would be better.